src/index.js

```diff
@@ -43,6 +44,9 @@ export default function algoliaAlexaAdapter (opts) {
   skill.handler = function(event, context) {
     const alexa = Alexa.handler(event, context);
     alexa.appId = alexaAppId;
+    if (languageStrings) {
+      alexa.resources = languageStrings;
```

Comment:
Ahh, yeah, I see what you're saying. I'm destructuring on line 18 to be the value that's provided in the `opts` object for the `languageStrings` key.